src/character.cpp

```diff
@@ -248,26 +248,22 @@ void Character::recalc_sight_limits()
         // Yes, I'm breaking the cap. I doubt the reality bubble shrinks at night.
         // BIRD_EYE represents excellent fine-detail vision so I think it works.
         if (has_trait("BIRD_EYE")) {
-            sight_boost = 13;
+            sight_boost_cap = 13;
```

Comment:
Hmm, this moves the cap but seems like it'd invalidate the boost thanks to the boost=cap being on the wrong side of an else.